impl/client/src/test/javascript/pentaho/data/filter/isGreater.spec.js

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright 2017 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "pentaho/data/filter/isGreater",
+  "pentaho/type/Context",
+  "pentaho/type/complex",
+  "./propertyUtils"
+], function(isGreaterFactory, Context, complexFactory, propertyUtils) {
+
+  "use strict";
+
+  /* global describe:true, it:true, expect:true, beforeEach:true*/
```

Comment:
I asked because it looks like to linter rules. Usually declaring `jasmine` env to `.eslintrc.json` is enough. I use VS code with linter plugin and it works fine for me. I don't use IDEA but I suppose so powerful tool should have something similar... because for now it looks really strange to declare this comment in each test source js files.